### PR TITLE
Add logs to recycler gibbing and broadcasting.

### LINKED ISSUE #25037: Admin Logs for Recycler Gib
## Description
There is no clear indication in the logs whenever an entity is gibed by an Emag'd recycler.

**Reproduction**
Search logs for "Recycler" or a player you know was gibed.

**Screenshots**
image

**Additional context**

### LINKED ISSUE #25556: Broadcasting isnt logged

### PR DESCRIPTION
**Pull request by nikthechampiongr**
fixes #25556 
fixes #25037 

## About the PR

This just adds logs for broadcasting and getting gibbed by a recycler.

## Why / Balance

Me admin. Me want this logs. Me beat logs with rock if I no know why passenger gibbed or who put the nword on all the monitors in the station.

## Technical details

The broadcast log uses the new DeviceNetwork log because I can find no other appropriate log type. The closest is Chat which announcements use but this makes no sense.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
ADMIN:
- add: Recyclers now leave logs when they gib people.
- add: People sending a broadcast in the communications console now leave logs.

### CODE REVIEW

**Comment by nikthechampiongr** [comment]
Why the hell did this request a review of all of you?

**Comment by nikthechampiongr** [comment]
> You should add ADMIN changelog. This is definitely something that Admins want to be aware of (the gibbing part especially).
> 
> > Why the hell did this request a review of all of you?
> 
> Don't worry about that, it's automatic.

I know I am just surprised it pinged that many.

**Comment by potato1234x** [comment]
no more moving the recycler into shuttle and gibbing tiders :despair:

File: Content.Server/Materials/MaterialReclaimerSystem.cs [hunk 1]
```diff
@@ -157,6 +160,7 @@ public override bool TryFinishProcessItem(EntityUid uid, MaterialReclaimerCompon
             SpawnChemicalsFromComposition(uid, item, completion, false, component, xform);
             _body.GibBody(item, true);
             _appearance.SetData(uid, RecyclerVisuals.Bloody, true);
+            _adminLogger.Add(LogType.Gib, LogImpact.Extreme, $"{ToPrettyString(item):victim} was gibbed by {ToPrettyString(uid):entity} ");
```
Comment (ShadowCommander) [inline Content.Server/Materials/MaterialReclaimerSystem.cs:163]: Have to move this above since the body doesn't exist anymore.

Comment (nikthechampiongr) [reply Content.Server/Materials/MaterialReclaimerSystem.cs]: Will do although I swear it worked just fine when I tested it.

**Review by ShadowCommander — APPROVED** [review]

**Comment by ShadowCommander** [comment]
Both logs show up when testing locally.

**Pull request merged**